docs/guidance/kubeflow-integration.md

```diff
@@ -0,0 +1,106 @@
+> Credit: This manifest refers a lot to the engineering blog ["Building a Machine Learning Platform with Kubeflow and Ray on Google Kubernetes Engine"](https://cloud.google.com/blog/products/ai-machine-learning/build-a-ml-platform-with-kubeflow-and-ray-on-gke) from Google Cloud.
+
+# Kubeflow: an interactive development solution
+
```

Comment:
Would be good to add a brief 1-2 sentences about what Kubeflow is, and why the user might care about this integration.  Even if it's redundant in the blog, many readers won't click on the blog.

Reply:
Will update it tomorrow.

Reply:
Updated https://github.com/ray-project/kuberay/pull/937/commits/52c78f15c6114a27b6d4298e876d16c6cb8c766d